script.js

```diff
@@ -288,3 +282,13 @@ edits.forEach((edit, index) => {
   });
   edit.append(element);
 });
+
+function handleDateChange(event) {
+  const preview = event.target.previousElementSibling.children[1];
```

Comment:
Place const input = event.target in starting of the function
and make it

```suggestion
  const preview = input.previousElementSibling.children[1];
```